src/lib/logger/logger.ts

```diff
@@ -1,8 +1,7 @@
 import pino from 'pino';
+import SonicBoom from 'sonic-boom';
```

Comment:
This should be on the `package.json`, pnpm is quite strict in terms of dependencies are not estrictly defined.
```
└─ pino@npm:6.11.2
   └─ sonic-boom@npm:1.4.0 (via npm:^1.0.2)
```

Reply:
Ignore this, we should not add `sonic-boom` dependency at all or use it as import, if Pino.js change that, then breaking change would be hard to fix.